Makefile

```diff
@@ -25,9 +25,17 @@ build:
 docker-build:
 	docker build -t $(DOCKER_NAME):$(GIT_TAG) -t $(DOCKER_NAME):latest . && echo ">> built $(DOCKER_NAME):$(GIT_TAG) and $(DOCKER_NAME):latest"
 
+.PHONY: buildah-f30
+buildah-f30:
+	buildah build-using-dockerfile -t $(DOCKER_NAME):$(GIT_TAG) -t $(DOCKER_NAME):latest -f Dockerfile.f30-mini . && echo ">> built $(DOCKER_NAME):$(GIT_TAG) and $(DOCKER_NAME):latest"
+
 .PHONY: docker-publish
 docker-publish:
 	docker push $(DOCKER_NAME):$(GIT_TAG) && echo ">> published $(DOCKER_NAME):$(GIT_TAG)"
 
+.PHONY: buildah-publish
```

Comment:
all this stuff seems very specific to the way you deploy containers. Presumably it would push an image an image with the same name as on docker hub to quay.io but using a different base image and different build process. It's kind of confusing. Should it at least use a tag suffix or something e.g. `$(GIT_TAG)-f30` to disambiguate it from the normal one?